staging-service/pom.xml

```diff
@@ -154,6 +154,11 @@
             <groupId>com.worldturner.medeia</groupId>
             <artifactId>medeia-validator-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.github.cafapi</groupId>
+            <artifactId>swagger-ui-annotation</artifactId>
+            <version>1.0.0-SNAPSHOT</version>
```

Comment:
Move this version to the dependency management section one level up, both the dependencies and the dependency management sections should be in alphabetical order.